### ISSUE TITLE
How to code completion

### ISSUE THREAD

**Issue by zraffaele** [AvaloniaUI/AvaloniaEdit#244]
Hi,
this editor support code complettion form assembly reference like visual studio?
Thanks

**Comment by Gillibald** [comment]
Have a look at: https://github.com/roslynpad/roslynpad

**Comment by zraffaele** [comment]
Yes i saw, but i want that feature in this editor because this editor has dark theme and Roslynpad haven't
